pkg/services/object/put/distributed.go

```diff
@@ -325,6 +325,7 @@ func (x placementIterator) iterateNodesForObject(obj oid.ID, f func(nodeDesc) er
 						return
 					}
 				}); err != nil {
+					wg.Done()
```

Comment:
A bit different (and incomplete) story.

Reply:
agree, just did not want two PRs after the same debug session in fact